extensions/smallrye-health/deployment/src/main/resources/dev-templates/embedded.html

```diff
@@ -0,0 +1,12 @@
+<a href="{frameworkRootPath}health" class="badge badge-light">
```

Comment:
I'm strongly opposed to `frameworkRootPath` that includes a `/` at the end, because it makes for very poor and confusing template usage like `{frameworkRootPath}health` where inevitably people will wonder if they need to include a `/` or not.

Can you change it to not include any last `/` and use `{frameworkRootPath}/health` in templates? This would make it consistent with the embedded path template parameter too.

Reply:
So, I initially had it like that, but.... to also make sure the case where httpRoot is / and nonApp is switch off, (so path is /) that we do not need to do 'if /' 
Now in all cases I know that I can just add my page. Cases include /, /httpRoot/, /httpRoot/nonAppRoot/, /nonAppRoot/

Do you think I should just add coded to always remove the last / ?

Reply:
Removed the /